app/src/controllers/searchController.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express'
+import { searchReservations } from './reservationController'
+import { searchRoles } from './roleController'
+import { searchShops } from './shopController'
+import { searchUser } from './userController'
+
```

Comment:
if we're importing everything from each controller, i think it would be better to just define all routes here, or add search route to all controllers